dev-pm.config.js

```diff
@@ -135,6 +135,12 @@ module.exports = {
             script: "docker compose up",
             group: ["demo-api", "demo"],
         },
+        {
+            name: "demo-api-generator",
+            script: "pnpm --filter comet-demo-api exec comet-api-generator generate --watch",
```

Comment:
I did not use the existing `api-generator:watch` script, as that does `rimraf --glob 'src/**/generated'` before starting the generator - and if demo-api and demo-api-generator are started at the same time, demo-api would have startup errors until api-generator re-generated.

An alternative could be to defer demo-api start until api-generator finished - but how?

Another alternative could be to ignore this problem and let demo-api startup just fail (and re-try). With #3789 it won't be too long.

Reply:
> Another alternative could be to ignore this problem and let demo-api startup just fail (and re-try).

I'm not a fan of this

Reply:
> An alternative could be to defer demo-api start until api-generator finished - but how?

It would be a nice dev-pm feature to wait for other processes to be "ready" (i.e. idle), but I have no idea how this can be achieved. Maybe the API Generator could write a file when it finishes one generation run?

Reply:
> It would be a nice dev-pm feature to wait for other processes to be "ready" (i.e. idle), 

We have the various methods waitFor supports.

> Maybe the API Generator could write a file when it finishes one generation run?

hm, that file would initially exist and make the waitFor pass.

Reply:
> Yes, but I can't specify other dev-pm processes directly

how should that work? There is no ready or idle state

> The rimraf call should of course delete this file as well.

yes, but initially it exists (and api-generator and api are started at the exact same time)

Reply:
> how should that work? There is no ready or idle state

Probably doesn't, but one can dream 😁